### DISCUSSION TITLE
Stand-alone actions

### DISCUSSION THREAD

**Discussion by adrianthedev** [avo-hq/avo#363] [Feature requests]
Avo has the ability to add actions to be run on one or more models.
It would be useful to have the ability to configure an action with the same DSL but to be run independently from models.

**Comment by calebuharrison** [comment]
Would love to see this functionality added!

**Comment by adrianthedev** [comment]
Added in `1.9.1`

**Reply by adrianthedev** [reply]
https://docs.avohq.io/1.0/actions.html#standalone-actions
